Index.html

```diff
@@ -13,206 +12,222 @@
         don&apos;t hestiate to contact me.</p>
-        <div class="grid-container">
-          <ul>
-            <li<a href="https://github.com/Zuz4n/"><img class="github"src="./images/Github.svg"></a></li>
-              <li<a href="www.linkedin.com/in/susana-glez-gdo"><img class="linkedin"src="./images/Linkedin.svg"></a></li>
-                <li<a href="www.linkedin.com/in/susana-glez-gdo"><img class="newent"src="./images/new.svg"></a></li>
-                  <li<a href="https://twitter.com/SuschenSu"><img class="twitter"src="./images/Twitter.svg"></a></li>
-                    <li<a href="siuxmicroverse@gmail.com"><img class="mail"src="./images/Mail.svg"></a></li>
+      <div class="grid-container">
+        <ul>
+          <li<a href="https://github.com/Zuz4n/"><img class="github" src="./images/Github.svg"></a></li>
+            <li<a href="www.linkedin.com/in/susana-glez-gdo"><img class="linkedin" src="./images/Linkedin.svg"></a></li>
+              <li<a href="www.linkedin.com/in/susana-glez-gdo"><img class="newent" src="./images/new.svg"></a></li>
+                <li<a href="https://twitter.com/SuschenSu"><img class="twitter" src="./images/Twitter.svg"></a></li>
+                  <li<a href="siuxmicroverse@gmail.com"><img class="mail" src="./images/Mail.svg"></a></li>
         </ul>
-        </div>
+      </div>
     </section>
 
     <section class="page1">
-      <h2 class="tituloMR">My Recent Works</h2>
-        <img class="vector"src="./images/Vector.svg">
-          <img class="yoga"src="./images/Img Placeholder.svg">
-      <h3 class="multi">Multi-Post Stories</h3>
-        <p class="parrafo1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-      <div class="list_programs">
-        <ul>
-          <li<a><img class="css"src="./images/css.svg"></a></li>
-            <li<a><img class="htm"src="./images/html.svg"></a></li> 
-              <li<a><img class="bot"src="./images/boot.svg"></a></li>
-                <li<a><img class="ruby"src="./images/ruby.svg"></a></li>
-        </ul>
-      </div>
-      <a target="_blank" class="b1" href="https://www.microverse.org/">See Project</a> 
-    
-      <div class="grid">
-      <div class="projects">
-        <h2 class="tituloPAPD">Profesional Art Printing Data</h2>
-          <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-        
-          <div class="list_programs2">
-          <ul>
-            <li<a><img class="htm"src="./images/html2.svg"></a></li> 
-              <li<a><img class="bot"src="./images/boot2.svg"></a></li>
-                <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+      <nav class="headerP2">
```

Comment:
[OPTIONAL]I suggest you change this from a `nav` tag to `div` or any other tag. Nav stands for navigation. It is very confusing having these in the middle of the page for screen readers.